tests/gold_tests/autest-site/min_cfg/storage.config

```diff
@@ -1,4 +1,4 @@
 # seems good enought for doing something for playing with.
 # not good for production
 # File must exist and must have this value in it
-storage 256MB
+storage 256M
```

Comment:
In the original issue, the user set the configured value as `1.5T` and ATS crashed. Is it better to set a similar float value in some of the autest?
Maybe a unit test is better for the ink_atoi64 function.